src/test/java/seedu/address/model/note/NoteTest.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.note;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+
+class NoteTest {
+    private static final String NOTE_CONTENT_1 = "note content 1";
+    private static final String NOTE_CONTENT_2 = "note content 2";
+    private static final String DUPLICATE_NOTE_CONTENT_1 = "note content 1";
+    private static final Note NOTE_1 = new Note(NOTE_CONTENT_1);
+    private static final Tag TAG_1 = new Tag("tag1");
+    private static final Tag TAG_2 = new Tag("tag2");
+    private final Set<Tag> tags;
+
+    public NoteTest() {
+        tags = new HashSet<>();
+        tags.add(TAG_1);
+        tags.add(TAG_2);
+    }
+
+    @Test
+    public void equals_sameNoteContents_returnTrue() {
+        Note note1 = new Note(NOTE_CONTENT_1);
+        Note note1Duplicate = new Note(DUPLICATE_NOTE_CONTENT_1);
```

Comment:
Wouldn't it be better to use `NOTE_CONTENT_1` to show that there will definitely not be a difference in the input to the two notes?

Reply:
Done